tests-functional/clients/status_backend.py

```diff
@@ -133,6 +133,20 @@ def init_status_backend(self, data_dir=USER_DIR):
         }
         return self.api_valid_request(method, data)
 
+    def _set_proxy_credentials(self, data):
+        if "STATUS_BUILD_PROXY_USER" in os.environ:
+            user = os.environ["STATUS_BUILD_PROXY_USER"]
+            password = os.environ["STATUS_BUILD_PROXY_PASSWORD"]
+
+            data["StatusProxyMarketUser"] = user
+            data["StatusProxyMarketPassword"] = password
+            data["StatusProxyBlockchainUser"] = user
+            data["StatusProxyBlockchainPassword"] = password
+
+            data["StatusProxyEnabled"] = True
+            data["StatusProxyStageName"] = "test"
+        return data
```

Comment:
Just a suggestion for early return pattern:
```suggestion
        if not "STATUS_BUILD_PROXY_USER" in os.environ:
            return data

        user = os.environ["STATUS_BUILD_PROXY_USER"]
        password = os.environ["STATUS_BUILD_PROXY_PASSWORD"]

        data["StatusProxyMarketUser"] = user
        data["StatusProxyMarketPassword"] = password
        data["StatusProxyBlockchainUser"] = user
        data["StatusProxyBlockchainPassword"] = password

        data["StatusProxyEnabled"] = True
        data["StatusProxyStageName"] = "test"
        return data
```